### DISCUSSION TITLE
VBET Validation

### DISCUSSION THREAD

**Discussion by lauren-herbine** [Riverscapes/riverscapes-tools#323] [VBET] [answered]
A place for people ("people" most likely just @shelbysawyer and I) working on VBET validation to discuss ideas and issues that come up. Also a place to list where we're working so that we're not overlapping.

**Comment by lauren-herbine** [comment]
@joewheaton @philipbaileynar in discussing how to get going on this, it was mentioned we should do this in places where we have or will have VBET for. I know we will have all of Montana and the Dakotas, but where else will VBET be/can I work on this? I've never been to the Dakotas and only have a smallish section of Montana I would consider myself "very familiar" with. I know you're doing some Kern plateau points @joewheaton as the TU project is there... are there other areas like this that I am not aware of that I could drop some points? Out west I have decent familiarity with the Truckee/Tahoe area, Umpqua River area, parts of the N. Yuba, parts of Sierra NF, and some SW Colorado. Any chance VBET is/will be in any of these places? Thanks!

**Reply by lauren-herbine** [reply]
Or on the off chance there are plans for east/midwest VBET...

**Reply by joewheaton** [reply]
@lauren-herbine feel free to do this anywhere you can be confident. I should not have said "somewhere you have visited".  Best if possible to do it in places we already have a VBET run and can quickly get a re-run for, but anywhere in Mississippi is in play and other places we have current contracts to do or have done are in play (e.g. Kern, or anything in Anabranch repo). @philipbaileynar can chime in too. He wants LOTS of points, and we want lots of diversity so I suspect he will be open to new runs, but they do take time and we have plenty of diversity in where VBET is already run.

**Comment by lauren-herbine** [comment]
@shelbysawyer right now I'm working in:

- 17010205
- 10070005
![image](https://user-images.githubusercontent.com/73319684/123691225-94a84a00-d812-11eb-8662-1512b9dc2b50.png)

- 10190010 (part of the MHFD, as per Philip's suggestion). Not as familiar with this area, so it may not be a full 100 pts.
Will edit with my last couple as I find appropriate watersheds

**Comment by lauren-herbine** [comment]
@shelbysawyer what are you doing about roads? Not dropping points on them? Theoretically these could flood/are in the historic valley bottom, but could now be considered a "terrace"?
Ex:
![image](https://user-images.githubusercontent.com/73319684/123693132-e7830100-d814-11eb-8fd1-36cef6ed3fc4.png)

Mostly wondering if dropping a point on this (because the roads should show up in the DEM) with a "very unlikely valley bottom extent"

**Reply by lauren-herbine** [reply]
@shelbysawyer another similar example/q:

![image](https://user-images.githubusercontent.com/73319684/123706312-5cf6cd80-d825-11eb-8bf3-8306b1bdd7fe.png)

If this road wasn't there, the northern part of the valley would likely also be floodplain. But because of the road, i wouldn't call that northern part "active." Would you call this inactive floodplain? Or just part of the plausible valley bottom?

**Reply by shelbysawyer** [reply]
> @shelbysawyer what are you doing about roads? Not dropping points on them? Theoretically these could flood/are in the historic valley bottom, but could now be considered a "terrace"?

I haven't been placing points on roads-- but I don't see why you shouldn't if you think they receive flooding. I can think of many roads in Maine, Pennsylvania and Maryland that I've personally seen flood. Here? None that come to mind... That being said, I think if you know the network or have a good idea of it from the RS context, you can use the DEM and aerial imagery to make reasonable observations about active vs inactive valley bottom.

I am personally being cautious about tagging anything as a terrace, partly because I haven't been to a lot of the places we are mapping VBET for and terraces can be a little tricky to identify (at least for me), particularly in areas of anthropogenically-induced valley bottom margins.

@joewheaton Will probably have more thoughts on this, but that's where I'm at for now.

**Reply by shelbysawyer** [reply]
> @shelbysawyer another similar example/q:

> If this road wasn't there, the northern part of the valley would likely also be floodplain. But because of the road, i wouldn't call that northern part "active." Would you call this inactive floodplain? Or just part of the plausible valley bottom?

Thanks for drawing out what you're working through-- I'm having a tough time distinguishing between certain categories at some points too. I have found it helpful to switch from starting with the categories to, instead, looking for places that I feel comfortable making a call about and then trying to match that to one of the options. If I am at an area that I really only feel comfortable dropping some active channel points and some upland points, that's what I do.

For your question about whether or not the area on the other side of the road is active, I feel like we can reasonably say it's inactive now, even though it was very likely active before people showed up and gave it a nice strip of gravel down the middle. That being said, I think it all (I can't see the DEM, but alluvial fans not included) falls within the valley bottom. 

That would leave me at _possible inactive valley bottom_ rather than the next option in that direction, _plausible valley bottom extent_, because I feel pretty comfortable saying it's in the valley bottom. If the DEM showed some topo variation that made me a little less sure in a few spots, I might throw a point down in those spots as _plausible valley bottom extent_. If there was something that felt very unlikely but wasn't quite "upland", I might throw down a point there too for _very unlikely valley bottom extent_.

**Reply by lauren-herbine** [reply]
@shelbysawyer Thanks for that last paragraph of explanation! That helps a lot. I think I have a better handle on "plausible valley bottom extent" now. And I guess a lot of my ponderings are just circling back to the question we've been asking a while- are we looking at pre-anthro disturbance conditions or current conditions?

**Reply by shelbysawyer** [reply]
@lauren-herbine I feel like I end up going in circles of overthinking around this same question.. maybe @joewheaton can refresh us on the best way to tease this concept apart. I'm making my observations based on what I believe is currently active/inactive floodplain _within_ the frame of the plausible valley bottom margin.

I also noticed that the observation recording categories aren't organized in the same order as the categories in the TOC. In the TOC, they are organized from most likely (active channel) down to least likely (very unlikely valley bottom extent). In the recording pop-up, they are mixed. It's not a big deal, but I think it can make the final category choice a little less intuitive to land on. I think there might already be an issue in progress to improve the observation recording process.

**Reply by joewheaton** [reply]
> > @shelbysawyer what are you doing about roads? Not dropping points on them? Theoretically these could flood/are in the historic valley bottom, but could now be considered a "terrace"?
> 
> I haven't been placing points on roads-- but I don't see why you shouldn't if you think they receive flooding. I can think of many roads in Maine, Pennsylvania and Maryland that I've personally seen flood. Here? None that come to mind... That being said, I think if you know the network or have a good idea of it from the RS context, you can use the DEM and aerial imagery to make reasonable observations about active vs inactive valley bottom.
> 
So, roads can certainly be in valley bottom. Remember, the main thing to validate here is in Valley Bottom versus outside, so don't stress the within categories as much. I agree with @shelbysawyer that typically the roads (if built up) are going to be inactive floodplain. 

> I am personally being cautious about tagging anything as a terrace, partly because I haven't been to a lot of the places we are mapping VBET for and terraces can be a little tricky to identify (at least for me), particularly in areas of anthropogenically-induced valley bottom margins.
> 
> @joewheaton Will probably have more thoughts on this, but that's where I'm at for now.

Good call on the terraces. With our definition of terrace, these will be really quite rare in most systems because they often do not exist. Inactive floodplains are far more common (disconnected as function of condition).

Don't force categories where you don't need them... you are not doing a cross section for real where you have to force every category. You're doing good sample points where you can make them. I find turning on and off the DEM/hillshade is really helpful for making calls as it helps me see what model will see.

Don't forget the non valley bottom points are just as important and really easy.

**Comment by philipbaileynar** [comment]
@lauren-herbine definitely keen to get some points on the plains in MHFD. Basically we have insufficient points for rivers of all sizes on very flat landscapes.

**Reply by lauren-herbine** [reply]
Ok, I will try my best there!

**Comment by shelbysawyer** [comment]
@lauren-herbine For now, I'm working in the Middle Bear (16010202) and the Little Logan-Bear (16010203). I'll update here when I land on a couple other HUCs to check out.

**Comment by lauren-herbine** [comment]
@shelbysawyer thoughts on this:

Would you consider only the current wetted width (what is apparent in the imagery) to be the estimated active channel? Or would you include the entire channel bottom area (outlined in purple)? I have never been here, so I don't know for sure. However, I know this is a spring snowmelt/flashy system. So, this channel bottom could be wetted to it's full width perennially. But then there's the current decade long drought we're in.... I've tried looking at aerial time lapse imagery from Google, but it hasn't been too enlightening. Additionally the DEM is too course here to be helpful. 

For now, I am calling the channel I can see the wetted width and the "channel bottom" _highly likely active floodplain_. This is probably something @joewheaton will say to just do active vs. upland. BUT I think it'll be good to discuss, as a lot of plains/ag/low relief systems will be like this.
![image](https://user-images.githubusercontent.com/73319684/123837301-ee1e8080-d8c7-11eb-8c00-5a5c69a8838a.png)

**Reply by shelbysawyer** [reply]
> Would you consider only the current wetted width (what is apparent in the imagery) to be the estimated active channel? Or would you include the entire channel bottom area (outlined in purple)?
> 
> For now, I am calling the channel I can see the wetted width and the "channel bottom" _highly likely active floodplain_.

@lauren-herbine I'll look for @joewheaton's response on this, but I would have done the same characterization that you did-- just calling the currently active channel (based on aerial imagery) _estimated active channel_ and the darker area that you've outlined in pink as _highly likely active floodplain_.

**Comment by shelbysawyer** [comment]
@lauren-herbine Can you give [this](https://youtu.be/A8ev5ETEaAY) video a look when you have time? I'm struggling with making calls in a few areas.

**Reply by lauren-herbine** [reply]
@shelbysawyer I have been struggling with the same exact thing on the plains east of Denver. I've been making sure I do a lot of active channel vs. upland points, and then in between attempting to use the DEM and slope analysis (admittedly not very helpful in this low relief topography) to discern slightly higher ground (and looking at distance from stream + vegetation) to try and get some semblance of variety on point type. Again, like you and I have both said, this goes back to current vs. historic condition. Historically, these wide, low relief valleys would be mostly floodplain, whereas today, the slope angle/landscape position says they should still be floodplain, but the conversion to agriculture and subsequent incision means there's less water, more regulation, and steeper banks to overbank. 

I know this doesn't necessarily help, but know that I am having the exact same questions! Perhaps @joewheaton will have some thoughts tonight.

**Comment by lauren-herbine** [comment]
Alrighty-ho @shelbysawyer [another question](https://youtu.be/srDT1NGa3lg) for you. Mostly pertaining to how you're handling ephemeral channels (and in my case: are they channels??). Thanks!

**Reply by shelbysawyer** [reply]
@lauren-herbine I watched your video, and I have to say, my grey areas are the same as yours. I'm curious about what @joewheaton thinks here? Seems like we are getting caught up in the same areas.

**Comment by shelbysawyer** [comment]
@lauren-herbine Are you tired of me yet? [Another question for you](https://youtu.be/fxXIlTASs34), when you have time. This is in South Dakota. Thank you!

**Reply by lauren-herbine** [reply]
[A reply](https://youtu.be/NxdZ2NgSEow). But I agree we are kind of going in circles with some of these things!

**Reply by shelbysawyer** [reply]
Hmm, I'm not getting audio on that video (sorry!). Don't worry about creating a whole new one.. I think I got the gist for using HAND?

**Comment by shelbysawyer** [comment]
@joewheaton I think it would be really helpful to both @lauren-herbine and I if you had time to make a quick video of yourself making some observations in the wide, low-angle valleys that we are struggling the most with. I know @philipbaileynar said we really need more points for channels of all sizes in low-angle terrain, but these areas are also the most difficult to judge. It would great to hear you go through your thought process. Thanks!

**Comment by joewheaton** [comment, marked as answer]
We met most of yesterday and went through this and ended up with close to 2800 points to run with. I'm going to consider this "Answered".

**Reply by shelbysawyer** [reply]
Agreed!
